README.md

```diff
@@ -54,6 +54,7 @@ Index
   * [Plugin Ruby API][12] - the API for plugins in Ruby
 * General
   * [Settings and Options][13] - specification of settings and options 
+  * [Unified Agent Layout][15] - specification of unified agent layout
```

Comment:
Small discrepancy on name - we are referring to it as AIO not UAL. Maybe say something else here so reader understands what the document is about - perhaps "Installation Layout on Disk"